### ISSUE TITLE
Revisit Travis build process for HHVM

### ISSUE THREAD

**Issue by initvector** [vanilla/vanilla#5201]
Vanilla's current Travis build config has a job for HHVM, along with a few different regular ole PHP versions. The problem with this is HHVM doesn't play by the same rules.

1. Vanilla has a HHVM config ini. It isn't used, because it's never loaded by HHVM. By the time we create the file, the daemon has already been loaded.
1. The build can't currently use `sudo` to manipulate services in HHVM. Since the service can't be restarted, the new config can't be loaded. This might be fixable with the proper Travis config.
1. The inability to load a custom INI means the default error logs are used by HHVM, which aren't being read. Vanilla tries to direct all relevant output to standard error or access logs and then read from them at the end of the build. With HHVM using its own logs, its errors are never populated in proper log. If it fails, it likely fails silently.

TL;DR: Need a way to make HHVM actually load its custom config settings in Travis.

**Comment by charrondev** [comment]
We no longer support HHVM in any way.
